lib/output-formatters/generate-raw-json.ts

```diff
@@ -10,20 +10,48 @@ export async function generateRawJson(
 ) {
   if (rawDeprecations.length === 0) {
     console.log(
-      "🎉 All deprecations are resolved, no raw JSON have to be generated"
+      "🎉 All deprecations are resolved, no raw JSON has to be generated"
     );
     return;
   }
 
   console.log("📝 Generating raw JSON");
 
-  const content = {
-    version: config.gitTag,
-    date: options.tagDate,
-    deprecations: rawDeprecations,
-  };
-  const json = JSON.stringify(content, null, 4);
-
   ensureDirExists(config.outputDirectory);
+
+  let existingData;
+  try {
+    const t = readFileSync(join(config.outputDirectory, `${config.gitTag}.json`));
+    existingData = JSON.parse(t as any);
+  } catch (e) {
+    existingData = false;
+  }
+  let content;
+  if (existingData) {
+    content = {
+      ...existingData,
+      deprecations: upsetrDeprecations(existingData.deprecations, rawDeprecations)
+    };
+  } else {
+    content = {
+      version: config.gitTag,
+      date: options.tagDate,
+      deprecations: rawDeprecations
+    };
+  }
+
+  const json = JSON.stringify(content, null, 4);
   writeFileSync(join(config.outputDirectory, `${config.gitTag}.json`), json);
 }
+
+function upsetrDeprecations(oldDeprecations: Deprecation[], newDeprecations: Deprecation[]): Deprecation[] {
+  const upsertedDeprecations = [...oldDeprecations];
+  upsertedDeprecations.forEach(d => upsetrDeprecation(upsertedDeprecations, d));
```

Comment:
Doesn't this have to be the new deprecations, in order to add them?

```suggestion
  newDeprecations.forEach(d => upsetrDeprecation(upsertedDeprecations, d));
```

Reply:
Also, it doesn't have to be done in this PR but we should also remove deprecations?
For example, deprecation is added, output is generated, deprecation is removed, output is generated and the deprecation is removed in the output.

The current implementation would update deprecations and add new deprecations, right?

Reply:
Right! Will fix it!